test/models/patient_test.rb

```diff
@@ -75,9 +78,34 @@ class PatientTest < ActiveSupport::TestCase
       assert @patient.valid?
       @patient.appointment_date = '2016-07-01'
       assert @patient.valid?
+      @patient.appointment_date = ( Date.today + 4 )
```

Comment:
Ah, here's a gotcha -- so one of the best/worst/best features of rails' testing library (Minitest) is that it's super good about isolating parts of your tests and running them in isolation, and then resetting everything afterwards. (This is one of the more domain-specific-y patterns of minitest.) I don't know much about non-ruby testing libraries so please forgive me if I'm pointing out something shared all over the place! A common pattern that you'll see goes like this:

```
describe 'appointment date behavior or some other cool feature' do
  before do
    # some setup stuff shared between tests in the describe block
    @patient = create :patient
  end

  it 'should be able to save an appointment' do
    @patient.appointment_date = '2016-11-10'
    @patient.save
    assert_equal '2016-11-10', @patient.appointment_date
  end

  it 'should not have an appointment on create' do
    assert_nil @patient.appointment_date
  end
end
```

In this paradigm, the `before` block gets executed before every test runs, so we run some common setup that's available to every test within the `describe`. (What's more, Minitest runs tests in a random order, so even if they did run in sequence here they'd still pass because `@patient` gets reset every time. And you'll notice that up top, on line 4, there's a `before` block for the entire test class - this gets run before every single `describe` block as well.

So in a nutshell, these lines aren't making it to the `pledge_summary` describe block. Chuck 'em in a `before` in the pledge summary describe and you should be in business! e.g.:

```
describe 'pledge summary' do
  before do
    @patient.update appointment_date: (Date.today + 4)
    # ... and so on
  end
  
  it 'should return sent and remaining pledge...'
     .....
```

sorry for the super long winded explanation -- hope it was more helpful than confusing!